src/app/game-state/battle.service.ts

```diff
@@ -301,7 +299,7 @@ export class BattleService {
       }
     }
     // it didn't match any, create a new enemyStack
-    this.enemies.push({enemy: JSON.parse(JSON.stringify(enemy)), quantity: 1});
+    this.enemies.push({enemy: enemy, quantity: 1});
```

Comment:
The reason I did a deep copy was so that I could damage the enemy without it damaging the enemy template for things like wolves and armies. Is that no longer an issue?

Reply:
AH, that helps with my understanding, I didn't think about the exact ramifications on the repo. I only tried one stack of mice during testing. Maybe I can add a parameter to keep deep copies to the repo so that the generated enemies can have shallow copies and reduce the performance loss. That should work.